src/crepe/system/PhysicsSystem.cpp

```diff
@@ -5,88 +5,95 @@
 #include "../api/Transform.h"
 #include "../api/Vector2.h"
 #include "../manager/ComponentManager.h"
+#include "../manager/LoopTimerManager.h"
+#include "../manager/Mediator.h"
 
 #include "PhysicsSystem.h"
 
 using namespace crepe;
+using namespace std::chrono;
 
 void PhysicsSystem::update() {
-	ComponentManager & mgr = this->mediator.component_manager;
+
+	const Mediator & mediator = this->mediator;
+	ComponentManager & mgr = mediator.component_manager;
+	LoopTimerManager & loop_timer = mediator.loop_timer;
 	RefVector<Rigidbody> rigidbodies = mgr.get_components_by_type<Rigidbody>();
-	RefVector<Transform> transforms = mgr.get_components_by_type<Transform>();
 
-	double gravity = Config::get_instance().physics.gravity;
+	float dt = std::chrono::duration<float>(loop_timer.get_scaled_fixed_delta_time()).count();
```

Comment:
physics hoort van mij niet chrono te includen

Reply:
omdat de LoopTimerManager een duration returned is dit makkelijk te converten naar secondes/millisecondes en als de looptimerManager waarde een float wordt is de aanroep alleen float dt = loop_timer.get_scaled_fixed_delta_time().count()